draft-ietf-core-coap-tcp-tls.md

```diff
@@ -138,7 +138,8 @@ reliable delivery, simple congestion control, and flow control.
 Some environments benefit from the availability of CoAP carried over reliable
 transports such as TCP or TLS. This document outlines the changes required to use
 CoAP over TCP, TLS, and WebSockets transports. It also formally updates {{RFC7641}}
-for use with these transports.
+for use with these transports, and RFC 6455 to extend well-known
+sockets (RFC 5785) to the ws and wss URI schemes.

```

Comment:
This is my "doh" moment today :-)

Of course, that should be "the well-known URI mechanism", going with the title of RFC 5785 (well-known locations is used within RFC 5785 and therefore would also be OK).

In an abstract, we don't make formal references(*), so the `{{RFC7641}}` actually should be `RFC 7641` (I did not make this change as it is unrelated and the RFC editor will make it anyway).

(*) RFC 7322 Section 4.3 "Abstract Section":

```
   Similarly, the Abstract should be complete in itself.  It will appear
   in isolation in publication announcements and in the online index of
   RFCs.  Therefore, the Abstract must not contain citations.
```